web/template/layout/mlpa-progress-indicator.gohtml

```diff
@@ -25,7 +25,26 @@
       </div>
     </summary>
     <div class="govuk-details__text">
-      -
+      {{ if eq (progressIndicatorContext .Indicator) "Donor identity confirmation" }}
```

Comment:
I think this is fine for now, but we should bear in mind that we might want to split out into separate templates as we do more of these.